src/commands/swap/swapConfirm.ts

```diff
@@ -36,16 +35,11 @@ async function makeQuestions(env: BlendEnvironment) {
         {
             type: 'list',
             name: 'from',
-            message: 'Address to reveal the hash from',
+            message: 'Address to confirm the swap from',
```

Comment:
Isn't _the_ redundant there?
```suggestion
            message: 'Address to confirm swap from',
```